tests/integration/Owncloud/OcisPhpSdk/DriveTest.php

```diff
@@ -87,4 +87,46 @@ public function testGetDriveRole(): void
             "Array contains not only 'SharingRole' items"
         );
     }
+
+    public function testCreateDriveInvite(): void
+    {
+        // At the time of writing, "stable" is major version 5 of ocis.
+        // This functionality works with major version 6.
+        // When ocis major version 6 has been released as "stable" then remove this test skip.
+        if (getenv('OCIS_VERSION') === "stable") {
+            $this->markTestSkipped(
```

Comment:
You could also add a separate test that only runs on "stable" and is skipped if not "stable".

That test can do the "invite", and expect an exception (or whatever happens, I think that the non-existent endpoint might cause a 404 "not found" response, and the invite will generate an exception because of that)

Advantages will be:
- we test that the SDK code does respond appropriately when the endpoint is not there
- when "stable" changes from being v5 of ocis to v6 of ocis, that test will suddenly fail and we will automatically realize that we need to adjust the test suite

Reply:
We need test code (could be in a separate test case) to test what happens on ocis v5,
Something similar to what was done in PR #229 
`EndPointNotImplementedException` needs to be thrown if the ocis version does not have the endpoint available.